configure

```diff
@@ -1896,12 +1889,9 @@ fi
 
 # Check for mmap support for huge pages and contiguous heap
 if sh ./runtest mmap-huge.c; then
-    has_huge_pages=true
```

Comment:
This variable is unused now. May be it worth adding it the final report table instead of removing?

Reply:
This was introduced in #297 and has never been used, but I don't think that's a case for its removal.

Reply:
Probably @damiendoligez can decide if this variable should be removed or reported at the end of configure run.